Yoast/a11y-speak
# Append a no-break space to repeated messages.

Fixes Safari+VoiceOver bug with identical, repeated, messages, appending a no-break space `\u00A0` character to the message string when it's the same as the previous message.

I've tried also with other characters like line separator (`\u2028`) and zero width no-break space (`\uFEFF`) but the no-break one looks more "safe" to me. The normal space (`\u0020`) doesn't work.

How to test:
- using Safari10+VoiceOver, open the Knowledge base search in the Help Center
- search for something that doesn't exist
- VoiceOver should announce "No results found."
- press again the Search button
- VoiceOver doesn't announce any message
- switch to this branch
- yarn link `a11y-speak` on Yoast SEO
- build the Yoast SEO js
- repeat the search, VoiceOver should announce "No results found."
- press again the Search button 
- verify Voiceover announces "No results found." again
- press again the Search button at your leisure, each time the search returns the message "No results found." VoiceOver should announce it correctly

<img width="992" alt="screen shot 2017-04-18 at 16 27 37" src="https://cloud.githubusercontent.com/assets/1682452/25136738/1a63c680-2456-11e7-90a7-5e3e4a540834.png">

If this gets merged, the temporary fix implemented on My Yoast should be removed, see https://github.com/Yoast/my-yoast/pull/224/commits/54b44b691cc10984173724fa7ccc03cdd1628c6a

Fixes #11
